pkg/kdump/kdump-view.jsx

```diff
@@ -283,55 +288,17 @@
         this.setState({ dialogObj });
     }
 
-    handleServiceDetailsClick(e) {
-        // only consider primary mouse button
-        if (!e || e.button !== 0)
-            return;
+    handleServiceDetailsClick() {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18435-20230309-172724-84b18f6a-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)